src/main/java/lotto/domain/LottoFactory.java

```diff
@@ -0,0 +1,26 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class LottoFactory {
+    private static final List<Integer> LOTTO_BASKET;
+    private static final int MIN_LOTTO_NUMBER = 1;
+    private static final int MAX_LOTTO_NUMBER = 45;
+
+    static {
+        ArrayList<Integer> lottoNumbers = new ArrayList<>();
```

Comment:
```suggestion
        List<Integer> lottoNumbers = new ArrayList<>();
```
[의존관계 역전 원칙](https://ko.wikipedia.org/wiki/%EC%9D%98%EC%A1%B4%EA%B4%80%EA%B3%84_%EC%97%AD%EC%A0%84_%EC%9B%90%EC%B9%99)

Reply:
조금 더 구체적으로 설명해주실 수 있을까요? 링크만 봐서는 `ArrayList`를 `List`로 바꿨을 때 어떤 이점이 있는지 잘 와닿지 않습니다.😥

Reply:
```
 ArrayList<Integer> lottoNumbers = new ArrayList<>();
```
의 구현체를 LinkedList 로 변경한다고 하면 어떤가요?